### PR TITLE
related to #798

### PR DESCRIPTION
**Pull request by Nowosad**

### CODE REVIEW

File: 05-geometry-operations.Rmd [hunk 1]
```diff
@@ -516,9 +516,9 @@ t = cast_all(multipoint)
...
 ```
 
-Geometry casting of simple features geometry column (`sfc`) and simple features objects works the same as for single geometries in most of the cases. 
+Geometry casting of simple features geometry column (`sfc`) and simple features objects works the same as for `sfg` in most of the cases. 
 One important difference is the conversion between multi-types to non-multi-types.
-As a result of this process, multi-objects are split into many non-multi-objects.
+As a result of this process, multi-objects of `sfc` or `sf` are split into many non-multi-objects.
 
 Table \@ref(tab:sfs-st-cast) shows possible geometry type transformations on simple feature objects.
 Each input simple feature object with only one element (first column) is transformed directly into another geometry type.
```
Comment (Robinlovelace) [inline 05-geometry-operations.Rmd]: I think this could be explained more clearly.

**Review by defuneste — COMMENTED** [review]
This is way better, specially L507. Let me try to improve a bit L524 (and see now, if I understand it correctly).  

> Each input simple feature object with only one element (first column) is transformed directly into another geometry type.

---> 

When a simple feature object with just one element is used in input it is transformed directly into another geometry type.


And L527

> You can see that, for example, when you cast a multipoint consisting of five pairs of coordinates into a point.

---> 

You can see that, for example, when you cast a multipoint consisting of five pairs of coordinates into a point (one pair of coordinates). 

Correct ?

**Comment by Robinlovelace** [comment]
To keep up momentum I'm going to merge this. We can see how it looks in the updated text and then decide if it fixes the issue. Any further comments welcome @defuneste. I think the table still could be explained better in terms of exactly what the inputs in the first column are. Any ideas on that? Far out idea is to create a small plot representing each input...

**Review by Robinlovelace — APPROVED** [review]

**Comment by defuneste** [comment]
https://github.com/Robinlovelace/geocompr/blob/main/05-geometry-operations.Rmd#L524-L527

This seems good to me and showed that I misunderstood the first version!  

I do not see how we can improve the table. An exercise on it could be an option but E6 is already a bit on it.

**Pull request merged**